brightwind/utils/utils.py

```diff
@@ -252,3 +254,39 @@ def validate_json(json_to_check, schema):
             print(f"Failed schema part: {error.get('schema_path')}\n")
     
     return data_is_valid
+
+
+def apply_scale_factor(data : Union[float, pd.Series, np.array],
```

Comment:
should this accept also dataframe? the scale_wind_speed function accepts also this type so the generic function called should do the same...please update here and on docstring